docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,44 +1,99 @@
 [[connect-to-elasticsearch]]
-== Connect Kibana with Elasticsearch
+== Adding data
 
-Before you can start using Kibana, you need to tell it which Elasticsearch indices you want to explore.
-The first time you access Kibana, you are prompted to define an _index pattern_ that matches the name of
-one or more of your indices. That's it. That's all you need to configure to start using Kibana. You can
-add index patterns at any time from the <<settings-create-pattern,Management tab>>.
+To start working with your data in {kib}, you can:
 
-TIP: By default, Kibana connects to the Elasticsearch instance running on `localhost`. To connect to a
-different Elasticsearch instance, modify the Elasticsearch URL in the `kibana.yml` configuration file and
-restart Kibana. For information about using Kibana with your production nodes, see <<production>>.
+* Use the File Data Visualizer or GeoJSON Upload feature to quickly load
```

Comment:
Do we really want to say `File Data Visualizer or GeoJSON Upload feature` before we tell them what these features are? I would pair down this list to:

- Load your data from a log file
- Index logs, metrics, events, and application data
- Connect Kibana to your Elasticsearch indices

Then get into the nitty gritty in the sections below.

Reply:
> Load your data from a log file

It might be worth mentioning CSV files here also as they are a very common use case.